packages/manager/modules/pci/src/components/project/warning-message/warning.constants.js

```diff
@@ -0,0 +1,10 @@
+export const WARNING_DATE = {
+  DATE_FORMAT: 'dd/MM/YYYY',
+  DB_CREATION_DATE: '01/02/2022',
+  LAST_DATE: '31/01/2022',
+  WARNING_END_DATE: '28/02/2022',
```

Comment:
Those dates must be coming from API, no ?

Reply:
No there is no API to provide this for now.